src/engine/common/enums.hxx

```diff
@@ -34,6 +35,8 @@ static Layer layersInActiveOrder[] = {Layer::BUILDINGS, Layer::UNDERGROUND, Laye
                                       Layer::ROAD,      Layer::POWERLINES,  Layer::FLORA,     Layer::ZONE,
                                       Layer::WATER,     Layer::TERRAIN};
 
+const static std::vector<Layer> layersToBeCross = {Layer::ROAD, Layer::WATER, Layer::FLORA};
```

Comment:
Add a comment what this is for and rename it to something more speaking, like "layersPowerlinesCanCross"